docs/devops-guide/manual.md

```diff
@@ -164,18 +164,34 @@ ln -s ../sites-available/jitsi.example.com jitsi.example.com
 ```
 
 ## Install Jitsi Videobridge
-Visit https://download.jitsi.org/jitsi-videobridge/linux to determine the current build number, download and unzip it:
+
+You can download binary packages for Debian/Ubuntu:
+* [stable](https://download.jitsi.org/stable/) ([instructions](https://jitsi.org/downloads/ubuntu-debian-installations-instructions/))
+* [testing](https://download.jitsi.org/testing/) ([instructions](https://jitsi.org/downloads/ubuntu-debian-installations-instructions-for-testing/))
+* [nightly](https://download.jitsi.org/unstable/) ([instructions](https://jitsi.org/downloads/ubuntu-debian-installations-instructions-nightly/))
+
+Or you can clone the Git repo and run the JVB from source using maven.
+
+Install JDK and Maven if missing:
+
 ```sh
-wget https://download.jitsi.org/jitsi-videobridge/linux/jitsi-videobridge-linux-{arch-buildnum}.zip
-unzip jitsi-videobridge-linux-{arch-buildnum}.zip
+apt-get install openjdk-8-jdk maven
 ```
 
-Install JRE if missing:
-```
-apt-get install openjdk-8-jre
+Clone source from Github repo:
+
+```sh
+cd /srv
+git clone https://github.com/jitsi/jitsi-videobridge.git
 ```
 
-_NOTE: When installing on older Debian releases keep in mind that you need JRE >= 1.7._
+Build the package:
+
+```sh
+export JVB_HOME="The path to your JVB clone."
```

Comment:
An explanation for this would be good: `JVB_HOME="The path to your JVB clone."`
So it should be said, that we switch the folder and now want to link to the folder where we cloned the Videobridge repo into.

Also where is this folder: `jitsi/jitsi-videobridge`
Should the user create it?

Reply:
There is an error here. The clone only contains the jitsi videobridge folder, which I created manually

Reply:
Can you please adjust that? You could export the content of `pwd` once you are in the directory.